### DISCUSSION TITLE
How to create / choose a custom theme?

### DISCUSSION THREAD

**Discussion by rootuser** [Power-Components/livewire-powergrid#824] [Q&A] [answered]
Hello,

I would like to use a custom theme on powergrid, but the documentation page has a bug:
the 'Theme' link here [https://livewire-powergrid.com/get-started/configure.html#_3-choose-your-theme](url) points to a non-existent page.

I had a look into the Powergrid documentation GitHub repository, but it seems the page is really missing (no 'custom-theme' page in https://github.com/Power-Components/powergrid-doc/tree/main/docs/table)

Perhaps someone can tell me how to work with themes, or, yet better, the documentation might be completed ; -)

Thanks! 
Regards, Tom

**Comment by dansysanalyst** [comment, marked as answer]
Doc updated: https://livewire-powergrid.com/table/custom-theme.html

**Reply by jemsk** [reply]
The v4 doc on how to edit is here. The marked as answer link does not work. @dansysanalyst 

https://v4.livewire-powergrid.com/table/custom-theme.html
